client/state/plugins/installed/reducer.js

```diff
@@ -33,6 +36,21 @@ export function isRequesting( state = {}, action ) {
 	}
 }
 
+/*
+ * Tracks the requesting state for installed plugins on a per-site index.
+ */
+export function isRequestingAll( state = {}, action ) {
+	switch ( action.type ) {
+		case SITES_PLUGINS_REQUEST:
+			return Object.assign( {}, state, { all: true } );
```

Comment:
Just a note: we don't seem to be needing to maintain this state subtree as an object. All we care about here is a boolean flag, so we might as well just return `true` or `false` instead of maintaining an object. Just don't forget to also set the default value of `state` to `false` too. That way you won't even need any `typeof` or any other check in the selector, and could just return the value.